doc/_static/css/custom.css

```diff
@@ -233,17 +233,104 @@ div.sk-landing-footer {
 }
 
 /** Scikit-learn buttons ***************************/
-a.sk-btn-orange {
+/* a.sk-btn-orange {
     font-size: 1.1rem;
     font-weight: 500;
-    background-color: #f99f44;  /* sk-orange-tint-1 */
+    background-color: #f99f44;  // sk-orange-tint-1
     color: black !important;
+} */
+
+a.sk-btn-orange {
+    font-size: 1.1rem;
+    font-weight: 500;
+    background-color: #f99f44; /* Original sk-orange-tint-1 */
+    color: #ffffff; /* Changed to white for better contrast */
+    border: 2px solid #f99f44; /* Border to match background color */
+    padding: 10px 20px; /* Added padding for better button size */
+    border-radius: 5px; /* Rounded corners for a modern look */
+    text-decoration: none; /* Remove underline */
+    transition: background-color 0.3s ease, color 0.3s ease; /* Smooth transition */
 }
 
 a.sk-btn-orange:hover {
-    background-color: #fcb575;  /* --sk-orange-tint-3 */
+    background-color: #ffffff; /* Fill with white on hover */
+    color: #f99f44; /* Text color changes to match the original background color */
+    border: 2px solid #f99f44; /* Border remains the same on hover */
+}
+
+
+a.sk-btn-orange:hover {
+    background-color: #fbe1ca;  /* --sk-orange-tint-3 */
 }
 
+
+.container {
+    max-width: 80%; /* Restrict container width for better readability */
+}
+
+.feature-title {
+    color: #f99f44; /* Darker text color for the title */
+    font-weight: 600; /* Bold font for title */
+    margin-bottom: 10px; /* Spacing below title */
+}
+
+.feature-text {
+    color: #555; /* Lighter grey for the description text */
+    font-size: 1rem; /* Regular font size for text */
+    margin-bottom: 0; /* Remove default bottom margin */
+}
+
+
+
+.hero-description {
+    text-align: center;
+    font-size: 20px;
+    margin-bottom: 40px;
+}
+
+
+.thin-line {
+    width: 100%;
+    height: 0.5px; /* Adjust the thickness of the line */
+    background-color: #12134b; /* Change the color as needed */
+    border: none; /* Ensures no border is applied */
+    margin-top: 50px;
+    margin-bottom: 50px;
+}
+
+
+.contributors-list {
```

Comment:
this is a really nice addition; the only drawback is the github avatars make the page much bigger

Reply:
Why is that a drawback?

Reply:
Ok, to be kept in mind if things start to get slow. Also, since we're using github API, maybe we would run into rate limits at some points?

Reply:
Ok, to be kept in mind if things start to get slow. Also, since we're using github API, maybe we would run into rate limits at some point?

Reply:
I would be very surprised as it's just one request when loading the page. many projects seem to show contributors like this without problems, I wouldn't worry about it